frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -61,10 +61,11 @@ const makeOCSRequest = (
   selectedData: NodeKind[],
   storageClass: StorageClassResourceKind,
   osdSize: string,
+  isEncrypted: boolean,
 ): Promise<any> => {
   const promises = makeLabelNodesRequest(selectedData);
   const scName = getName(storageClass);
-  const ocsObj = getOCSRequestData(scName, osdSize);
+  const ocsObj = getOCSRequestData(scName, osdSize, undefined, isEncrypted);
```

Comment:
You can change the order of parameters here which will avoid you from passing `undefined`

Reply:
Noted